Automattic/wp-calypso
# DIFM Lite: Initialise the 4th page from the site category

#### Changes proposed in this Pull Request

This PR will be merged into the parent PR: https://github.com/Automattic/wp-calypso/pull/59895.

* Initializes the 4th-page section for the Website Content step from the site category selected in the Design Picker step. This mimics the behaviour of the current form.

| Design Category | Title of the 4th page |
|----|----|
|Professional Services | Services |
|Local Services | Services |
|Creative Arts | Portfolio |
|Restaurant | Menu |
|Blog | Blog |


#### Testing instructions



* Purchase DIFM Lite by proceeding through the flow at `/start/do-it-for-me`.
* In the design picker step, note down the category selected.
* After checkout, on the Website Content step, check the title of the 4th-page section. Confirm that it matches the title from the table above.
